### PR TITLE
Implement new designs for deployment tool & inline connection errors

### LINKED ISSUE #186: Implement new designs of deployment tool
__What should we do?__

Implement new designs for the deployment tool: 

https://app.zeplin.io/project/5e149dc7ef8eaa51b52af327/screen/5e71dd9ddd3148b4e1ec7813

__Why should we do it?__

It looks better 🖌 

Related to https://github.com/camunda/camunda-modeler/issues/1708

Child of https://github.com/zeebe-io/zeebe-modeler/issues/183
Child of https://github.com/zeebe-io/zeebe-modeler/issues/200

### PR DESCRIPTION
**Pull request by barmac**
This PR implements new designs for deployment tool. It also inlines connection errors as we do not have a global connection feedback anymore.

Closes #186

### CODE REVIEW

**Comment by barmac** [comment]
@nazlikaya Can you have a look at this?

File: client/src/plugins/zeebe-plugin/deployment-plugin/ui/Radio.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  return (
+    <React.Fragment>
+      <div className={
+        classNames('form-group', 'form-inline')
+      }>
+        <label htmlFor={ fieldName }>{ label }</label>
+        <div className="form-check-inline">
+          {
+            values.map((child) => {
+              const id = 'radio-element-' + toKebabCase(child.label);
```
Comment (pinussilvestrus) [inline client/src/plugins/zeebe-plugin/deployment-plugin/ui/Radio.js:44]: Well, I'm hungry now 🥙

Comment (barmac) [reply client/src/plugins/zeebe-plugin/deployment-plugin/ui/Radio.js:44]: Gotta invent currywurstCase

**Comment by barmac** [comment]
Let's call it a WIP now. I just discovered one issue:

Screenshot 2020-04-02 at 10 47 11

**Comment by barmac** [comment]
OK fixed. A bunch of screenshots
Screenshot 2020-04-02 at 11 02 10
Screenshot 2020-04-02 at 11 02 21
Screenshot 2020-04-02 at 11 02 08

**Comment by oguzeroglu** [comment]
I did some manual tests and looking good 👍The fix works too!

I'm not approving the PR as we're waiting to here from Nazli. If she gives ok I think we can merge this.

**Comment by pinussilvestrus** [comment]
I also briefly tested it and it worked as expected 👍

**Comment by pinussilvestrus** [comment]
Does this also include an improved input validation logic? I remember we had to separate PRs for design and logic in the Camunda Modeler. Or do we simply don't need any extra improvements here?

**Comment by barmac** [comment]
> Does this also include an improved input validation logic? I remember we had to separate PRs for design and logic in the Camunda Modeler. Or do we simply don't need any extra improvements here?

It's just the designs, the validation logic will come next, in a separate PR.

**Comment by andreasgeier** [comment]
@andreasgeier mentioned me to find that in my list. Take on the design review task from Nazli.

**Comment by andreasgeier** [comment]
**Design Review** 

Please fix style and alignment of the triangle of the tooltip (see [Zeplin](https://zpl.io/aR8dD5n)):

- shape and its border needs correction
- alignment has to be centered to warning icon, consider spacing between them

Screen Shot 2020-04-06 at 11 48 07

**Comment by oguzeroglu** [comment]
I realized we have the same issue in Camunda Modeler as well, so we'll have to apply the fix there too.

**Comment by pinussilvestrus** [comment]
> I realized we have the same issue in Camunda Modeler as well, so we'll have to apply the fix there too.

Can you maybe open an issue there as well?

**Comment by barmac** [comment]
> **Design Review**
> 
> Please fix style and alignment of the triangle of the tooltip (see [Zeplin](https://zpl.io/aR8dD5n)):
> 
> * shape and its border needs correction
> * alignment has to be centered to warning icon, consider spacing between them
> 
> Screen Shot 2020-04-06 at 11 48 07

This will be fixed via https://github.com/camunda/camunda-modeler/pull/1765

**Comment by barmac** [comment]
I pulled here the tooltip arrow fix as well as fixed the labels pointed out today. @andreasgeier please feel free to have a look again.

Regarding the mixing auth method & deployment target thing, I created a separate issue as it is outside of scope of this issue: https://github.com/zeebe-io/zeebe-modeler/issues/203

**Comment by andreasgeier** [comment]
✅  Design Review: Passed. Tooltip arrow looks fine.

**Pull request merged**